src/main/webapp/WEB-INF/tags/instructor/course/activeCoursesTable.tag

```diff
@@ -33,10 +33,18 @@
         <tr>
             <td id="courseid${i.index}">${activeCourse.courseId}</td>
             <td id="coursename${i.index}">${activeCourse.courseName}</td>
-            <td class="align-center">${activeCourse.sectionNum}</td>
-            <td class="t_course_teams align-center">${activeCourse.teamNum}</td>
-            <td class="align-center">${activeCourse.totalStudentNum}</td>
-            <td class="align-center">${activeCourse.unregisteredStudentNum}</td>
+            <td id="course-stats-sectionNum-${i.index}">
+                <a class="course-stats-link-${i.index}" oncontextmenu="return false;" href="${activeCourse.href}">Show</a>
```

Comment:
I think it is necessary since these links do not work with the right click context menu